external-import/s3/src/s3.py

```diff
@@ -354,11 +357,12 @@ def process(self):
                 content = data["Body"].read()
                 self.helper.log_info("Sending file " + o.get("Key"))
                 fixed_bundle = self.fix_bundle(content)
-                self.helper.send_stix2_bundle(bundle=fixed_bundle, work_id=work_id)
-                self.helper.log_info("Deleting file " + o.get("Key"))
-                self.s3_client.delete_object(
-                    Bucket=self.s3_bucket_name, Key=o.get("Key")
-                )
+                if fixed_bundle:
+                    self.helper.send_stix2_bundle(bundle=fixed_bundle, work_id=work_id)
+                    self.helper.log_info("Deleting file " + o.get("Key"))
+                    self.s3_client.delete_object(
+                        Bucket=self.s3_bucket_name, Key=o.get("Key")
+                    )
 
```

Comment:
Thanks for improving this part ! 

May I suggest an additional information ?

```suggestion
                if fixed_bundle:
                    self.helper.send_stix2_bundle(bundle=fixed_bundle, work_id=work_id)
                    self.helper.log_info("Deleting file " + o.get("Key"))
                    self.s3_client.delete_object(
                        Bucket=self.s3_bucket_name, Key=o.get("Key")
                    )
                 else:
                    self.helper.log_info("No content to ingest") # EXAMPLE

```

Reply:
Done! Thanks